docs/eggs/configuration.md

```diff
@@ -84,36 +94,40 @@ checkAll: false
     - Defaults value is determined by Semantic Versioning rules.
 - unlisted:
     - Should people be able to find this module/version on the gallery?
-    - Defaults to false.
+    - Defaults to `false`.
 - repository:
     - A link to your repository.
-    - Defaults to null.
+    - Defaults to `null`.
 - files:
     - All the files that should be uploaded to nest.land. Supports file globbing.
+    - Will throw an error if some files are not found.
 > Do not use `./**/*` for the `files` field! This has been known to cause errors in the publishing process.
 - ignore:
     - All the files that should be ignored when uploading to nest.land. Supports file globbing.
     - For more details, please refer to [#eggignore](#eggignore).
 - checkFormat:
     - Automatically format your code before publishing to the blockchain.
-    - Defaults to false.
+    - Accepts a boolean or a string (user defined command).
+    - Defaults to `deno fmt` if `true`.
```

Comment:
```suggestion
    - Executes `deno fmt` if `true`.
```
I'm not 100% sure if "defaults to" would be the proper wording here. I may be misunderstanding though.

Reply:
Yes it's not so obvious
```sh
eggs publish --check-format "some command" # will run `some command`
eggs publish --check-format # will run `deno fmt`
```